pkgs/by-name/sc/scopehal-apps/package.nix

```
@@ -0,0 +1,100 @@
+{
+  stdenv,
+  breakpointHook,
+  fetchFromGitHub,
+  cmake,
+  git,
+  lib,
+  pkg-config,
+  libsigcxx,
+  gtkmm3,
+  cairomm,
+  yaml-cpp,
+  catch2,
+  glfw,
+  libtirpc,
+  liblxi,
+  glew,
+  libllvm,
+  libdrm,
+  elfutils,
+  libxcb,
+  zstd,
+  libxshmfence,
+  wrapGAppsNoGuiHook,
+  xcbutilkeysyms,
+  systemd,
+  shaderc,
+  vulkan-headers,
+  vulkan-loader,
+  vulkan-tools,
+  glslang,
+  spirv-tools,
+  ffts,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scopehal-apps";
+  version = "unstable-2024-08-22";
+
+  src = fetchFromGitHub {
+    owner = "ngscopeclient";
+    repo = "scopehal-apps";
+    rev = version;
+    hash = "sha256-sKrxElL4XRHqCPHsH8Q/2pJeQwnMex0YAQMU5I4FEVA=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  strictDeps = true;
  nativeBuildInputs = [
```
By enabling strictDeps, you can make sure `buildInputs` and `nativeBuildInputs` are right.
This will help fix crosscompilation.